zipline/pipeline/loaders/blaze/core.py

```diff
@@ -547,20 +562,25 @@ def from_blaze(expr,
                odo_kwargs=None,
                missing_values=None,
                no_deltas_rule='warn',
-               no_checkpoints_rule='warn'):
+               no_checkpoints_rule='warn',
+               apply_deltas_adjustments=True,):
     """Create a Pipeline API object from a blaze expression.
 
     Parameters
     ----------
     expr : Expr
         The blaze expression to use.
+    apply_deltas_adjustments : bool, optional
```

Comment:
the arguments in the docstring should be in the same order as the function signature